scripts/artifacts-hashes.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Looks like the other scripts are calling `#!/usr/bin/env bash`. We should probably stay consistent.